cmd/create/network/cmd.go

```diff
@@ -0,0 +1,157 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/network"
+	helper "github.com/openshift/rosa/pkg/network"
+	"github.com/openshift/rosa/pkg/ocm"
+	bsOpts "github.com/openshift/rosa/pkg/options/network"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+type NetworkSpec struct {
```

Comment:
I have a preference for not repeating the package name when naming types. For example, if this were accessed externally it would look like `network.NetworkSpec`, when `network.Spec` also conveys the same information.

This is also suggested by the Go blog if you search https://go.dev/blog/package-names

Reply:
There are a few other instances of functions/structs/etc in this PR that the same comment applies to as well. I won't tag all of them to keep the review a bit cleaner, but I'll leave it up to your discretion.